### PR TITLE
Add message of new node added to bucket

### PR DESCRIPTION
**Pull request by r4f4ss**
Implements #191 

Add message: "Node added to bucket"

### CODE REVIEW

File: p2p/discover/portal_protocol.go [hunk 1]
```diff
@@ -553,8 +553,10 @@ func (p *PortalProtocol) processOffer(target *enode.Node, resp []byte, request *
 	}
 
 	p.Log.Trace("<< ACCEPT/"+p.protocolName, "id", target.ID(), "accept", accept)
-	p.table.addFoundNode(target, true)
-
+	isAdded := p.table.addFoundNode(target, true)
+	if isAdded {
+		log.Info("Node added to bucket", "protocol", p.protocolName, "node", target.IP(), "port", target.UDP())
```
Comment (GrapeBaBa) [inline p2p/discover/portal_protocol.go:558]: I would like not to use info level for these logs, shouldn't we also log a waring if adding failed?

File: p2p/discover/portal_protocol.go [hunk 2]
```diff
@@ -748,7 +759,11 @@ func (p *PortalProtocol) processNodes(target *enode.Node, resp []byte, distances
 		return nil, err
 	}
 
-	p.table.addFoundNode(target, true)
+	isAdded := p.table.addFoundNode(target, true)
+	if isAdded {
+		log.Info("Node added to bucket", "protocol", p.protocolName, "node", target.IP(), "port", target.UDP())
+	}
+	log.Info("Node added to bucket", "protocol", p.protocolName, "node", target.IP(), "port", target.UDP())
```
Comment (GrapeBaBa) [inline p2p/discover/portal_protocol.go:766]: duplicate

File: p2p/discover/portal_protocol.go (hunk 1 above)
Comment (r4f4ss) [reply p2p/discover/portal_protocol.go]: > I would like not to use info level for these logs

I moved from "info" to "debug", that is shown in loglevel 4

> shouldn't we also log a waring if adding failed?

"warn" is shown in loglevel 3, that is the default, and fills the screen with messages, I suggest to use "debug" loglevel as well using the message "Node added to replacements list"

File: p2p/discover/portal_protocol.go (hunk 2 above)
Comment (r4f4ss) [reply p2p/discover/portal_protocol.go]: fixed

**Review by GrapeBaBa — APPROVED** [review]

**Pull request merged**